server/error.go

```diff
@@ -55,14 +57,21 @@ var (
 // ErrorResponse is a standard way to return errors to the client
 //
 // Specs: http://bsvalias.org/99-01-recommendations.html
-func ErrorResponse(w http.ResponseWriter, code, message string, statusCode int) {
+func ErrorResponse(w http.ResponseWriter, req *http.Request, code, message string, statusCode int, log *zerolog.Logger) {
+	if log == nil {
+		log = logging.GetDefaultLogger()
+	}
+
 	srvErr := &paymail.ServerError{Code: code, Message: message}
 	jsonData, err := json.Marshal(srvErr)
 
 	if err != nil {
+		log.Debug().Msgf("%d | %s | %s | %s | %s", http.StatusInternalServerError, req.RemoteAddr, req.Method, req.URL, message)
```

Comment:
Additionally we could add those params in contextual fields and maybe some additional marker like Str("logger", "http-response")

Reply:
Done